tools/mason/MasonExample.chpl

```diff
@@ -289,7 +289,7 @@ private proc runExampleBinary(projectHome: string, exampleName: string,
 
   const exampleResult = runWithStatus(command, true);
   if exampleResult != 0 {
-    throw new owned MasonError("Mason failed to find and run compiled example: " + exampleName + ".chpl");
+    throw new owned MasonError("Example has not been compiled: " + exampleName + ".chpl\nTry running: mason build --example " + exampleName + ".chpl\n\t or: mason run --example " + exampleName + ".chpl --build");
```

Comment:
@klakho0400 - error message is good, but can you split this line up across multiple lines?